timezone_converter/search_view.py

```diff
@@ -0,0 +1,34 @@
+from difflib import get_close_matches
+from typing import List
+
+from timezone_converter.helper import Helper
+
+
+class SearchView(Helper):
+    def __init__(
+        self,
+        search: str,
+    ) -> None:
+        self.search = search
+
+    def _search_and_sort(self, search: str) -> List[str]:
+        timezones: List[str] = get_close_matches(
+            search,
+            self.timezone_translations,
+        )
+
+        if len(timezones) > 0:
+            timezones.sort()
+
+        # If the search term has a corresponding timezone, return it instead
+        if search in timezones:
+            timezones = [search]
+
+        return timezones
+
+    def print_search_results(self) -> int:
+        timezones = self._search_and_sort(self.search)
+        self._print_with_rich(
+            f"Found {len(timezones)} {'timezone' if len(timezones) == 1 else 'timezones'}: {timezones}",  # noqa: E501
+        )
```

Comment:
Here the output is just a list, and I would prefer just strings, but we can't just do a simple join to be able to take advantage of Rich's syntax highlighting, so we have to do something like:

```py
', '.join(map(lambda tz: f'"{tz}"', timezones))
```

This will output something like this (second line is the actual one, for comparison):

![image](https://user-images.githubusercontent.com/45886240/135761473-5642f097-af59-4dfb-86f4-7795b997a14f.png)

And lastly, `f-strings` are nice, but they sometimes don't play nice with short line length limits, so it would be better to just use `.format`. With all this changes, this would end up looking like this:

```py
        self._print_with_rich(
            'Found {} {}: {}'.format(
                len(timezones),
                'timezone' if len(timezones) == 1 else 'timezones',
                ', '.join(map(lambda tz: f'"{tz}"', timezones))
            )
        )
```